apps/www/components/Marketing/Team.js

```diff
@@ -13,26 +13,27 @@ import SectionContainer from './Common/SectionContainer'
 import Link from 'next/link'
 
 const EmployeeLink = ({ employee, children }) => (
-  <Link href={`/~${employee.user.slug || employee.user.id}`} {...styles.link}>
+  <Link href={`/~${employee.userId}`} {...styles.link}>
     {children}
   </Link>
 )
 
 const Team = ({ employees, title, description }) => {
   const [colorScheme] = useColorContext()
+  const threeMarketingEmployees = employees.slice(1).sort(() => 0.5 - Math.random()).slice(0, 3)
```

Comment:
Hmm, this will probably lead to a hydration error because the set will be different on the server and client